src/client/apis/CsioRTCClient.js

```diff
@@ -136,27 +135,6 @@ class CsioRTCClient {
     }
     return '';
   }
-  detectScreenShareExtenstion() {
-    const updateExtensionDetail = (isInstalled) => {
-      localStorage.setItem('csioExtension', JSON.stringify(isInstalled));
-    };
-    const extensionURL = `chrome-extension://${__addon_id__}/logo_48_48.png`;
-    let xhr = new XMLHttpRequest();
-    xhr.open('GET', extensionURL, true);
-    xhr.onload = function(e) {
-      if (xhr.readyState === 4) {
-        if (xhr.status === 200) {
-          updateExtensionDetail(true);
-        } else {
-          updateExtensionDetail(false);
-        }
-      }
-    };
-    xhr.onerror = function(e) {
-      updateExtensionDetail(false);
-    };
-    xhr.send(null);
-  }
```

Comment:
No need to remove this. I think we can still point to the extension for installing if its not installed.

Reply:
This extension installed check was for old extension. With new implementation for CSIO Jisti based chrome extension - we check if extension is installed or not in runtime, and ask user to download extension if it is not exist.